profile/2018-03-01-hybrid.json

```diff
@@ -0,0 +1,163 @@
+{
+    "info":{
+        "name":"2018-03-01-profile",
+        "description":"Profile definition targeted for hybrid applications that could run on azure stack 1807 version and azure cloud"
+    },
+    "resource-manager": {
+        "microsoft.authorization": {
+			"2016-09-01": [
+				"locks"
+			],
+			"2015-07-01": [
+				"operations",
+				"permissions",
+				"roleAssignments",
+				"roleDefinitions"
+			],
+			"2016-12-01": [
+				"policyAssignments",
+				"policyDefinitions"
+			],
+			"2015-07-01-preview": [
```

Comment:
Not this one either.

Reply:
It seems azure team has left out specifying this providerOperations resource type for this api-version. This is not critical resource type and i am removing it from the profile map